components/email/src/Email.svelte

```diff
@@ -54,36 +54,53 @@
   export let is_starred: boolean;
   export let show_contact_avatar: boolean;
   export let clean_conversation: boolean;
+  export let show_expanded_email_view_onload: boolean;
 
+  // Track `show_expanded_email_view_onload` value in expandThreadOnLoad.
+  // This lets us limit expanding thread once, onload
+  $: expandThreadOnLoad = show_expanded_email_view_onload || false;
```

Comment:
@pooja169usp what happens if we simply use `show_expanded_email_view_onload` with a default value of `false`? Does it fire multiple times? 🤔

Reply:
Hey @AaronDDM , yes it not only fires multiple times, but also goes gets initialized back to the value that was passed in. So, if I pass `show_expanded_email_view_onload =true`, and after setting expanded to true, set it to false, it gets reset reactively.